crates/ra_syntax/src/parsing/reparsing.rs

```diff
@@ -289,7 +289,6 @@ impl IntoIterator<Item=i32> for Foo {
             "n next(",
             9,
         );
-        do_check(r"use a::b::{foo,<|>,bar<|>};", "baz", 10);
```

Comment:
This test is no longer eligible for block-based reparsing because the new error-recovery behavior no longe results in a single parse tree covering the `{…}` block.